code/modules/clothing/glasses/glasses.dm

```diff
@@ -10,7 +10,7 @@
 	var/prescription = 0
 	var/prescription_upgradable = 0
 	var/see_darkness = 1
-	var/HUDType = 0
+	var/HUDType = null

```

Comment:
Not sure if this is necessary, when the only thing that handles hude types is glasses/hud (on equipped and stuff)--same goes for anything that goes on your head. Specific reasons for this?